ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -225,13 +228,17 @@ describe('AdminInstitutionEditSpec - edit mode', () => {
 
     await simulateComponentChange(wrapper, findRTDropdown(wrapper), InstitutionMembershipRequirement.ADDRESSES);
 
-    expect(findRTAddressError(wrapper).exists()).toBeFalsy();
+    expect(findRTAddressError(wrapper)).toBeTruthy();
```

Comment:
The Save button tooltip always informed the user that the addresses should not be empty.

This test change is because we're using a new way of detecting the error text - checking the Save button tooltip itself instead of additional text on the page (which is no longer present)